python3/totoro_agent/brain/bomb_tracker.py

```diff
@@ -2,17 +2,45 @@
 Used for tracking lifecycle of bombs and explosion areas
 """
 
+from ..utils.util_functions import get_blast_zone
 
 class BombTracker:
     def __init__(self):
         self.bombs = []
 
     def update(self, game_state):
         # TODO Will
+        # Get own and enemy id
+        own_id = str(game_state['connection']['agent_number'])
```

Comment:
can just do own_id = game_state["player_id"] because this is available in game_state after enemy_tracker execution